app.py

```diff
@@ -0,0 +1,29 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
```

Comment:
This needs to be moved to a .env file or some untracked file. Please create a new secret key and move appropriately.
Use `os.getenv("MY_API_SECRET_KEY")` to access the secret key from .env file